moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -20,7 +20,7 @@ import DeleteIcon from "@material-ui/icons/DeleteOutlined";
 import EditIcon from "@material-ui/icons/EditOutlined";
 
 import { makeStyles } from "@material-ui/core/styles";
-import { getSessionDatabaseData, getHighestRole, useUser } from "src/auth/user";
```

Comment:
these methods are no longer needed here because we are no longer allowing admins to edit other users' comments.